src/RaceCar/raceCar.js

```diff
@@ -0,0 +1,36 @@
+import { GIDE_MESSAGE } from "../Constants/constant.js";
+import Output from "../Output/Output.js";
+import randomGenerator from "../Utils/randomGenerator.js";
+
+const moveRaceCar = (car) => {
+  const randomNum = randomGenerator();
+  if (randomNum >= 4) {
+    car.position += 1;
+  }
+};
+
+/**
+ *
+ * @param {string[]} carList
+ * @param {int} tryNumber
+ * @returns {string[]} winnerCars
+ * @summary 자동차 리스트와 시도 횟수를 받아서 가장 멀리간 자동차를 반환한다.
+ */
+const raceCar = (carList, tryNumber) => {
+  const raceCarList = carList.map((car) => {
+    return {
+      name: car,
+      position: 0,
+    };
+  });
```

Comment:
```suggestion
  const raceCarList = carList.map((car) => ({name: car, position: 0}));
```
이렇게 `return`문을 생략할 수 있을 것 같아요!

Reply:
앗 그렇네요 ㅋㅋㅋㅋㅋㅋㅋ 짧게 줄일 수 있는 부분인 것 같습니다.